src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,52 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Duke {
+    private static final String MESSAGE_HI = "Hello! I'm Duke\n" + "What can I do for you?\n" + "!help for Command List\n";
+    private static final String MESSAGE_BYE = "Bye. Hope to see you again soon!\n";
+
+    protected static boolean isOnline;
+
+    private static void startDuke(){
+        isOnline = true;
+        System.out.println(Logo.logo +Logo.divider + MESSAGE_HI + Logo.dividerWithoutNewLine);
+    }
+
+    private static void endDuke(){
+        System.out.println(Logo.divider + MESSAGE_BYE + Logo.divider + Logo.bye);
+    }
+
+    public static void main(String[] args) {
+        startDuke();
+        Scanner in = new Scanner(System.in);
+        ArrayList <Task> list = new ArrayList<>();
+        ListManager listManager = new ListManager(list);
+        FileManager fileManager = new FileManager(listManager);
+        fileManager.covertStringToTask();
+        while(isOnline) {
+            String userInput = in.nextLine().toLowerCase().trim();
+                if (userInput.startsWith("!")) {
```

Comment:
perhaps a magic string here? maybe change to a named constant in case you are going to use it somewhere else...